python/phylanx/ast/physl.py

```diff
@@ -514,6 +533,9 @@ def lazy(self, args=()):
         if not PhylanxSession.is_initialized:
             PhylanxSession.init(1)
 
+            for func in PhySL.functions:
+                func.compile()
+
```

Comment:
Will the functions be compiled also if the user explicitly calls `PhylanxSession.Init()` (possibly before any of the decorated functions are being seen)?

Reply:
Not if the user manually initializes Phylanx in between defining Phylanx functions. I just pushed a fix.

Reply:
Thanks!